src/program/db/db_functions.py

```diff
@@ -403,74 +349,66 @@ def _get_item_type_from_db(item: "MediaItem") -> str:
     from program.media.item import MediaItem
     with db.Session() as session:
         if item._id is None:
-            return session.execute(select(MediaItem.type).where((MediaItem.imdb_id==item.imdb_id) & (MediaItem.type.in_(["show", "movie"])))).scalar_one()
-        return session.execute(select(MediaItem.type).where(MediaItem._id==item._id)).scalar_one()
+            return session.execute(select(MediaItem.type).where(MediaItem._id==item._id)).scalar_one()
 
-def _store_item(item: "MediaItem"):
-    from program.media.item import Episode, Movie, Season, Show
-    if isinstance(item, (Movie, Show, Season, Episode)) and item._id is not None:
-        with db.Session() as session:
-            item.store_state()
-            session.merge(item)
-            session.commit()
-            logger.log("DATABASE", f"{item.log_string} Updated!")
-    else:
-        with db.Session() as session:
-            _check_for_and_run_insertion_required(session, item)
+def _get_item_from_db(session: Session, item_id: int) -> "MediaItem":
+    from program.media.item import MediaItem, Movie, Show, Season, Episode
 
-def _imdb_exists_in_db(imdb_id: str) -> bool:
-    from program.media.item import MediaItem
-    with db.Session() as session:
-        return session.execute(select(func.count(MediaItem._id)).where(MediaItem.imdb_id == imdb_id)).scalar_one() != 0
+    if not item_id:
+        raise AttributeError("Missing Item ID")

```

Comment:
The method isnt callable without item_id, this is redundant

Reply:
ah forgot to remove, was using for testing to make sure nothing was getting through :)